VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -294,8 +285,7 @@ protected void onOnline() {
     @Override
     protected void onOffline() {
         super.onOffline();
-        showErrorMessage(getString(R.string.no_connectivity),
-                getString(R.string.network_not_connected), false);
+        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
```

Comment:
I had not previously reviewed / seen the Error Handling page. I'll review these details to understand what is going on here. Thanks @BenjiLee and @brianguertin for the details here!